### DISCUSSION TITLE
REQUETS - Add Search Term Impression Share for Sponsored Products and Brands in API

### DISCUSSION THREAD

**Discussion by maverick-marc** [amzn/ads-advanced-tools-docs#244] [Ideas]
I'm manually downloading this report weekly in the ad console reports section and would be great if I can access this data via the API.
I know this isn't available via the API yet but it would be super helpful to have it added to the API.  It's important because we can then understand which terms we lost Impression Share on and Rank so we can prioritize where to spend our ad dollars better.

**Comment by elizafarley** [comment]
Thanks for this feedback @maverick-marc! I will pass this on the product team. 

In may not be exactly what you're looking for, but we do have a [topOfSearchImpressionShare](https://advertising.amazon.com/API/docs/en-us/release-notes/index#top-of-search-impression-share-available-through-the-amazon-ads-api) metric available for SP and SB reports that you may find helpful.

**Reply by maverick-marc** [reply]
The only issue I have with that report is it provides data only at a Target level and not a Search Term level.  The SP & SB Impression Share Report provides me with the Impression Share and Rank at a specific Search Term Level.  For example a single Search Term may be receiving spend from 10 different Targets across my campaigns, in which case getting data at a Target Level is not as useful as a Search Term Level especially with Amazon introducing semantic targeting where an EXACT target doesn't really mean EXACT anymore.

**Reply by maverick-marc** [reply]
Do you have plans to add the Customer Search Term level topOfSearchImpressionShare metric for SP and SB reports soon?

**Reply by maverick-marc** [reply]
@elizafarley perhaps my interpretation is incorrect.  By "keyword level" for "topOfSearchImpressionShare" does this mean a Customer Search Term vs a Target?  Example: "garlic press" in EXACT match is the target and the Customer Search Term is "garlic presses"  then in this case by "keyword" is it referring to "garlic presses" vs the target "garlic press" in EXACT match which because of semantic match could catch a variety of Customer Search Terms.   Ideally I can get the metric specific to a single word (Customer Search Term) and not a group of words.

**Reply by maverick-marc** [reply]
Having both topOfSearchImpressionShare and also overall SearchImpressionShare would allow me as an advertiser to better understand how important the topofsearch placement really is.  For example for some categories it's not as important because customers will always compare at least 7-10 items (imagine a high priced category) whereas for other categories which are more impulse buys (products that are a lower priced category) being in the topofsearch placement may be much more important.  I won't be able to know unless I gather the data together and right now I am doing it manually from the ad console to create that report which funny enough doesn't allow me to generate a topOfSearchImpressionShare report in the ad console

**Reply by elizafarley** [reply]
Thanks for supplying all this context! I've passed this on the product team. I'll come back and update this thread if we are able to add these metrics.

**Reply by Nilanth** [reply]
@elizafarley Do we have any updates on the Sponsored Products Search Term Impression Share report API? We also waiting for this API.

**Comment by nrcjea001** [comment]
Any updates here? Specifically looking for impression share across all search terms, not just top of search. It is available in the console, but not sure if available in the API

**Reply by BrainInTheJar** [reply]
Doesn't appear Amazon has any intention of API-support of STIS anytime soon.  
Even if they just added the STIR & STIS metrics to the Sponsored Products Report via API would be helpful.

**Comment by harrisonpy** [comment]
and ?

**Comment by AlexWho3000** [comment]
Anyone reading us?

**Reply by harrisonpy** [reply]
I'm afraid not. 😂
